python_bindings/requirements.txt

```diff
@@ -8,5 +8,5 @@
 numpy
 scipy
 pillow
-imageio
-pybind11
+imageio==2.4.1
```

Comment:
Adding comments here (from the PR description) would be useful for future maintainers